flutter_appcenter_bundle/lib/flutter_appcenter_bundle.dart

```diff
@@ -57,6 +57,16 @@ class AppCenter {
     });
   }
 
+  /// Track error
+  static Future trackErrorAsync(String message,
+      [Map<String, String>? properties]) async {
+    await _methodChannel.invokeMethod('trackError', <String, dynamic>{
+      'message': message,
+      'properties': properties,
+      // Support `ErrorAttachmentLog`?
```

Comment:
do we think support `ErrorAttachmentLog` is necessary?

Reply:
Wonder if it might be relevant to support passing stack trace from Dart?

```dart
runZonedGuarded<Future<void>>(() async {
  runApp(App());
}, (error, stackTrace) => AppCenter.trackErrorAsync(error, stackTrace));
```